### PR TITLE
Snpsites

### PR DESCRIPTION
**Pull request by aunderwo**
## PR checklist
Removing a hard-coded arg that should be user-specified if required
Closes #XXX 

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `<SOFTWARE>.version.txt` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
    - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd`

### CODE REVIEW

File: software/snpsites/main.nf [hunk 1]
```diff
@@ -29,7 +29,7 @@ process SNPSITES {
     script:
     def software = getSoftwareName(task.process)
     """
-    snp-sites -c \\
+    snp-sites \\
         $alignment \\
         > filtered_alignment.fas
```
Comment (drpatelh) [inline software/snpsites/main.nf:34]:
```suggestion
    snp-sites \\
        $alignment \\
        $options.args \\
        > filtered_alignment.fas
```

Comment (drpatelh) [reply software/snpsites/main.nf:34]: We would need `options.args` defined here to provide `-c`?

Comment (aunderwo) [reply software/snpsites/main.nf:34]: -c could be provided in the `options.args`, yes.
I am not sure I understand the question

Comment (drpatelh) [reply software/snpsites/main.nf:34]: Need to add `$options.args` to the command as I suggested so we can pass `-c` to the module as an optional argument now instead.

**Review by drpatelh — APPROVED** [review]
LGTM!

File: software/snpsites/main.nf (hunk 1 above)
Comment (aunderwo) [reply software/snpsites/main.nf]: Sorry - I thought I'd done that. 🤦 
That was always my intention

**Pull request merged**